src/neuroglancer/datasource/state_share.ts

```diff
@@ -73,8 +73,9 @@ export class StateShare extends RefCounted { // call it a widget? no because it
             }, responseJson)
           .then((res) => {
             const stateUrl = new URL(res);
-            stateUrl.protocol = protocol; // copy protocol in case it contains authentication type
-            const link = `${window.location.origin}/#!${stateUrl}`;
+            const stateUrlWithoutProtocol = stateUrl.toString().split(stateUrl.protocol)[1];
```

Comment:
Split is kind of an odd way to handle this -- instead could just use substring(stateUrl.protocol.length)

Reply:
Yes, made this change